doc/nrf/libraries/networking/wifi_credentials.rst

```diff
@@ -64,7 +64,7 @@ The library has the following limitations:
 API documentation
 *****************
 
-| Header file: :file:`include/net/wifi_credentials.h`
+| Header file: :file:`include/zephyr/net/wifi_credentials.h`
```

Comment:
From the PR what I understand is both the wifi_credentials and wifi_mgmt_ext libraries are removed. So, shouldn't the both the .rst file also be removed? could you confirm @maxd-nordic .

Reply:
Yes, we are planning to remove this PR from 2.9.0 scope.

Reply:
can confirm that these libraries are moved to Zephyr.